test/src/sdk-harness/test_artifacts/tx_contract/src/main.sw

```diff
@@ -1,20 +1,53 @@
 contract;
 
 use std::{
+    b512::B512,
+    bytes::Bytes,
     inputs::{
         Input,
+        input_amount,
         input_count,
+        input_message_data,
+        input_message_data_length,
+        input_message_msg_id,
+        input_message_nonce,
+        input_message_recipient,
+        input_message_sender,
         input_owner,
         input_pointer,
+        input_predicate,
+        input_predicate_data_length,
+        input_predicate_data_pointer,
+        input_predicate_length,
         input_type,
+        input_witness_index,
     },
+    logging::log,
     outputs::{
         Output,
+        output_amount,
         output_count,
         output_pointer,
         output_type,
     },
-    tx::*,
+    tx::{
+        Transaction,
+        tx_gas_limit,
+        tx_gas_price,
+        tx_id,
+        tx_maturity,
+        tx_receipts_root,
+        tx_script_bytecode_hash,
+        tx_script_data_length,
+        tx_script_data_start_pointer,
+        tx_script_length,
+        tx_script_start_pointer,
+        tx_type,
+        tx_witness_data,
+        tx_witness_data_length,
+        tx_witness_pointer,
+        tx_witnesses_count,
```

Comment:
At this point, it must be better to just use `*` import for `inputs::` and `tx::`.